### PR TITLE
Remove double border footer

### PR DESCRIPTION
**Pull request by jordesign**
re: #196 styles to use margin instead of padding on borders on .entry-footer and .site-footer so the border's don't combine to form a 2px section

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1730,7 +1731,8 @@ body:not(.twentyseventeen-front-page) .entry-header {
 
 #comments {
 	border-bottom: 1px solid #eee;
-	padding: 2em 0 0.5em;
+	padding: 0 0 0.5em;
+	margin: 2em 0 0 0;
```
Comment (ntwb) [inline style.css]: Use shorthand `margin: 2em 0 0;`, not `margin: 2em 0 0 0;`

**Review by ntwb — CHANGES_REQUESTED** [review]

**Comment by jordesign** [comment]
This PR has become a bit muddied once merged with current master. It seems possibly less needed?

At any rate - it now separates .entry-footer and .site-footer as referenced in #196 
image

File: style.css [hunk 2]
```diff
@@ -3287,6 +3282,7 @@ article.panel-placeholder {
 		font-size: 14px;
 		font-size: 0.875rem;
 		line-height: 1.6;
+		margin-top:3em;
```
Comment (ntwb) [inline style.css]: Expected a single space after the colon

**Review by ntwb — CHANGES_REQUESTED** [review]

**Comment by melchoyce** [comment]
Except for when comments exist on pages, I don't think we actually need the bottom border. I'd rather have the top border on the footer be the divider instead:

screen shot 2016-10-12 at 9 55 41 pm

**Comment by jordesign** [comment]
I reckon even on a page _with_ comments it isn't needed.

Before:
image

After:
image

Only strange outlier seems to be if comments are on and we have post-paging. But I think it still looks alright.
image

File: style.css (hunk 2 above)
Comment (jordesign) [reply style.css]: this is fixed now

File: style.css [hunk 3]
```diff
@@ -926,7 +926,6 @@ a:hover .nav-title,
 }
 
 .post-thumbnail a img {
-	-webkit-backface-visibility: hidden;
```
Comment (laurelfulford) [inline style.css:929]: This was added to fix #212 - is there a reason it's being removed here?

**Comment by laurelfulford** [comment]
This fix looks good @jordesign! Added a question about an unrelated code change, but once that's figured out this should be good to go :)

**Comment by jordesign** [comment]
There was no good reason for that backface visibility being removed - i may have been in the wrong branch. I've popped it back in.

**Pull request merged**